cypress-shared/support/shopper-location/outputvalidation.js

```diff
@@ -13,4 +15,26 @@ export default {
     long: '80.2707',
     city: 'Chennai',
   },
+  franceDetails: {
+    country: 'FRA',
+    postalCode: '93200',
+    city: 'Saint-Denis',
+    state: 'IDF',
+  },
+  canadaDetails1: {
```

Comment:
Cross verify once that we are using all this properties
If some of them are not used pls remove